apps/projects/app/components/TabbedView/TabbedView.js

```diff
@@ -10,19 +10,11 @@ const StyledTabbedView = styled.div`
 `
 
 export default class TabbedView extends PureComponent {
-  state = {
-    activeIndex: 0,
-  }
-
-  selectTabIndex(activeIndex) {
-    this.setState({ activeIndex })
-  }
-
   render() {
     const children = React.Children.map(this.props.children, child => {
       return React.cloneElement(child, {
-        activeIndex: this.state.activeIndex,
-        onSelectTab: this.selectTabIndex.bind(this),
+        activeIndex: this.props.activeIndex,
```

Comment:
Why? Does really the parent component need to have info about the state of this child? In my opinion it is better to encapsulate the behaviour inside the component and don't take it out